ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/question_answering/SentenceHighlightingQATranslator.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.algorithms.question_answering;
+
+import static org.opensearch.ml.engine.algorithms.question_answering.QAConstants.*;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.jetbrains.annotations.NotNull;
+import org.opensearch.ml.common.input.MLInput;
+import org.opensearch.ml.common.output.model.ModelTensor;
+import org.opensearch.ml.common.output.model.ModelTensors;
+import org.opensearch.ml.engine.algorithms.question_answering.sentence.DefaultSentenceSegmenter;
+import org.opensearch.ml.engine.algorithms.question_answering.sentence.Sentence;
+import org.opensearch.ml.engine.algorithms.question_answering.sentence.SentenceSegmenter;
+
+import com.google.gson.Gson;
+
+import ai.djl.huggingface.tokenizers.Encoding;
+import ai.djl.huggingface.tokenizers.HuggingFaceTokenizer;
+import ai.djl.modality.Input;
+import ai.djl.modality.Output;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.translate.ServingTranslator;
+import ai.djl.translate.TranslatorContext;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * Translator for sentence highlighting question answering model.
+ *
+ * <p>Expected model output format:
+ * The model should output binary predictions for each sentence, where:
+ * - 0 indicates a relevant sentence (that answers the question)
+ * - 1 indicates a non-relevant sentence
+ * 
+ * This format can be customized by overriding the isRelevantPrediction method.
+ */
+@Log4j2
+@Getter
+@Builder
+public class SentenceHighlightingQATranslator implements ServingTranslator {
+    /**
+     * Default relevance value that indicates a sentence is relevant.
+     * By default, 0 means relevant and 1 means not relevant.
+     */
+    private static final long RELEVANT_VALUE = 0L;
```

Comment:
isn't it 0 indicates a relevant sentence? why is this 0L? can you update the comment?

Reply:
Using 0L(updated to 1L) makes it immediately clear to other developers that this is a long value, improving code readability.